cli.py

```diff
@@ -166,6 +166,10 @@ def fix_up(self):
     def extract(self):
         logging.info('Extracting scheduler module objs: %s', ' '.join(self.mod_objs))
         self.make(stage = 'collect', plugsched_tmpdir = self.tmp_dir, plugsched_modpath = self.mod_path)
+
+        self.plugsched_sh.cp(sym_vers,      self.work_dir, force=True)
```

Comment:
`sym_vers` is missing from this method

Reply:
Beter wrap these two lines in a function, so it looks more clear why they're put here